bundles/FileExplorerBundle/public/js/startup.js

```diff
@@ -0,0 +1,43 @@
+pimcore.registerNS("pimcore.fileexplorer");
+
+pimcore.fileexplorer = Class.create({
+    initialize: function () {
+        document.addEventListener(pimcore.events.pimcoreReady, this.pimcoreReady.bind(this));
+    },
+
+    pimcoreReady: function(e) {
+        const user = pimcore.globalmanager.get('user');
+        var perspectiveCfg = pimcore.globalmanager.get("perspective");
+
+        var toolbar = pimcore.globalmanager.get('layout_toolbar');
+
+        if (
+            user.admin &&
+            perspectiveCfg.inToolbar("extras") &&
+            perspectiveCfg.inToolbar("extras.systemtools") &&
+            perspectiveCfg.inToolbar("extras.systemtools.fileexplorer")
+        ) {
+            const index = toolbar.extrasMenu.items.keys.indexOf('pimcore_menu_extras_system_info');
+            const systemInfoMenu  = toolbar.extrasMenu.items.items[index];
+
+            systemInfoMenu.getMenu().add(
+                {
+                    text: t("server_fileexplorer"),
+                    iconCls: "pimcore_nav_icon_fileexplorer",
```

Comment:
We should move the css parts to the bundle too, so maybe move the icons to its own icons.css in the bundle